src/index.js

```diff
@@ -4,15 +4,9 @@ import { arrayWithLength } from './arrayWithLength.js'
 import { instanceOfComponent } from './instanceOfComponent.js'
 import { mutuallyExclusive } from './mutuallyExclusive.js'
 
-const customPropTypes = {
-    arrayWithLength,
-    instanceOfComponent,
-    mutuallyExclusive,
-}
+proptypes.arrayWithLength = arrayWithLength
```

Comment:
Why is `proptypes` in lowercase here but in camelCase in the default export (`propTypes`)?